### ISSUE TITLE
New NCA Facility: Elko

### ISSUE THREAD

**Issue by jilladams** [department-of-veterans-affairs/va.gov-cms#13654]
## Intake
- [x] What triggered this runbook? (Flag in CMS, Help desk ticket, Product team, VHA Digital Media)
Trigger: Flag

- [ ] Link to associated help desk ticket (if applicable)
Help desk ticket: <insert_help_desk_link>

- [ ] Name of submitter (if applicable)
Submitter: <insert_name>

- [ ] If the submitter is an editor, send them links to any relevate KB articles for the NCA Facility product.
KB articles: <insert_kb_article_links>

- [x] Link to facility in production:
Facility CMS link: https://prod.cms.va.gov/nca-facilities/locations/elko
Facility API ID: nca_943

### New NCA Facility
**Note: If the help desk is waiting on information from the facility staff or editor, add the "Awaiting editor" flag to the facility with a log message that includes a link to this ticket. Remove the flag when the ticket is ready to be worked by the Facilities team. Be sure to preserve the current moderation state of the node when adding or removing the flag.**
[@TODO: DRAFT FOR HELP DESK AND DEV STEPS]

#### Help desk steps
- [ ] Notify an editor to set an operating status so the facility will get pushed to the Facility API
- [ ] Verify when the status is added            

#### Drupal Admin steps
- [ ] Edit the facility node: remove the `New facility` flag, save the node

**Comment by omahane** [comment]
@davidconlon @mmiddaugh Do either of you know who is the editor of this or any NCA facility? The CMS is no help in this regard as the only status updates have been made by @davidconlon.

**Comment by davidconlon** [comment]
@omahane Actually I'm the only editor. NCA isn't in the CMS.

**Comment by jilladams** [comment]
Marking blocked based on Slack thread here: https://dsva.slack.com/archives/C0FQSS30V/p1684931438357459, and by new ticket #13816 for Dave/Michelle to review the runbooks.

**Comment by davidconlon** [comment]
I just removed the flag.
